universal-application-tool-0.0.1/test/app/BaseBrowserTest.java

```diff
@@ -165,4 +167,30 @@ protected void removeQuestionsToProgram(String programName, String... questions)
       assertThat(browser.$("#questionBankQuestions").$("li").textContents()).contains(question);
     }
   }
+
+  protected void fillInput(String name, String text) {
+    browser.$("input", withName(name)).fill().with(text);
+  }
+
+  protected String getInputValue(String name) {
+    return browser.$("input", withName(name)).first().value();
+  }
+
+  protected void fillTextArea(String name, String text) {
+    WebElement element =
+        browser.getDriver().findElement(By.cssSelector("textarea[name=" + name + "]"));
+    element.clear();
+    element.sendKeys(text);
+  }
+
+  protected String getTextAreaString(String name) {
```

Comment:
nit: rename to `getTextAreaValue` to match input?

Reply:
I wish! This doesn't get the value of the text area but returns the rendered html string, so I wanted to keep them different.